sphinx/Infinitude_of_Primes.rst

```diff
@@ -5,26 +5,28 @@ This article gives a formal statement of Infinitude of Primes.  For Wikipedia's
 description, see
 `Infinitude of Primes <https://en.wikipedia.org/wiki/Prime_number#Infiniteness>`_.
 
-Informally statement
```

Comment:
This is still formatted as a reST code block, which will not compile. The informal statement of the theorem can be done with plain text, e.g. "The theorem states that there are infinitely many primes."